### PR TITLE
'element' alias for 'elements'

### PR DESCRIPTION
**Pull request by ellenmobify**
Status: **Ready for Review** 
Owner: Ellen
Reviewers: 
## Changes
- Refactored `elements` method
- Adds `element` that behaves in the same way as `elements`
## Jira Tickets:
- https://mobify.atlassian.net/browse/CSOPS-1527
## Todos:
- [ ] +1
### Feedback:

_none so far_

### CODE REVIEW

**Comment by qa-derrick** [comment]
@donnielrt wanna have a look at this too? :+1: from me

File: README.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ All of these assertions can take an optional `msg` parameter to output a custom
 
 [ ] syntax denotes an optional parameter. 
 
-`elements([msg])`
+`elements([msg]) / elements([msg])`
```
Comment (donnielrt) [inline README.md]: Was this repeated intentionally?

Edit: I guess one should say `element`, eh?

Comment (ellenmobify) [inline README.md]: @donnielrt Good eye! Fixed it.

**Comment by donnielrt** [comment]
:+1: from me

**Pull request merged**